src/App.tsx

```diff
@@ -16,8 +18,20 @@ import {
   CREATE,
   HOME,
 } from './config/path';
+import { store } from '@/app/store.ts';
+import { initSDK } from '@/app/appSlice.ts';
 
 function App() {
+  const { connector } = useAccount();
+  const { chain } = useNetwork();
+
+  useEffect(() => {
+    if (!connector) {
+      return;
+    }
+    store.dispatch(initSDK({ connector }));
+  }, [connector, chain]);
```

Comment:
Here to watch what happens in the Metamask extension.
 - Disconnecting the wallet
 - Changing wallet
 - Changing chain

All should be supported = UI should react accordingly.